app/models/profile.rb

```diff
@@ -0,0 +1,5 @@
+class Profile < ApplicationRecord
+  validates :name, presence: true, uniqueness: true
```

Comment:
Why does a Profile has an unique name?

Reply:
Because it's a person model classification.